modules/MotionParser.cpp

```diff
@@ -410,7 +433,35 @@ bool MotionParser::convertBool(const string& command, const string& stringParame
     }
   }
 
+  // IMU設定(IS)の場合、"START"ならtrue（開始）、"STOP"ならfalse（停止)に変換
+  if(command == "IS") {
+    if(param == "start") {
+      return true;
+    } else if(param == "stop") {
+      return false;
+    } else {
+      cout << "'START' か 'STOP'を入力してください" << endl;
+      return true;
+    }
+  }
+
   // ここまでに条件を満たしていなかった場合は、デフォルト値としてtrueを返す
   cout << "convertBool関数の処理の対象外です: '" << command << endl;
   return true;
 }
+
+bool MotionParser::convertMode(const string& stringParameter)
```

Comment:
なんのモードなのかを記載はして欲しいかなー。
```suggestion
bool MotionParser::convertIMURotationMode(const string& stringParameter)
```

Reply:
2つしかないなら toggle か switch でもいいかもね。将来的に3つ以上なら convert がいいと思う。
それか、set◯◯ で相対と絶対どっちかに設定する関数を別に作って、その中で色々やるのもいいかもね。
ここらは裁量になるけど。

Reply:
convertRotationModeToBool()とかにしときます